app/models/changeset.rb

```diff
@@ -54,6 +54,19 @@ def author_names
     commits.map(&:author_name).uniq
   end
 
+  def reviewers
+    @reviewers ||= begin
+      cache_key = [self.class, "reviewers", repo, previous_commit, commit]
+
+      Rails.cache.fetch(cache_key) do
+        pull_requests.
+          flat_map { |pr| GITHUB.pull_request_reviews(repo, pr.number) }.
+          map(&:user).
+          uniq
```

Comment:
Looks like `uniq` is still allowing duplicate users. I think we might want to make this `uniq(&:login)`.